src/rev.cpp.in

```diff
@@ -72,6 +72,22 @@ const byte _openttd_revision_modified = ${REV_MODIFIED};
  */
 const byte _openttd_revision_tagged = ${REV_ISTAG};
 
+/**
+ * To check compatibility of BaNaNaS content, this version string is used.
+ * It should never contain things like "beta", but only the release version
+ * we are compatible with.
+ */
+const char _openttd_content_version[] = "12.0";
+
+/*
+ * Patchpacks can introduce their own content version, which is an extension
+ * to the above variable. See RequestContentList() in network_content.cpp for
+ * more information.
+
+const char _openttd_content_version_patchpack[] = "0.1.2";
```

Comment:
I think the example in RequestContentList is enough. IMO keep rev.cpp.in clean :)